proposals/0000-typed-metadata.md

```diff
@@ -147,11 +151,28 @@ import mypack.MyModule;
 function doThing() { ... }
 ```
 
+&nbsp;
+
 ### Untyped Metadata
 
 If the Haxe compiler encounters a metadata entry it cannot type, its behavior is currently an [Unresolved Question](0000-typed-metadata.md#unresolved-question). 
 
-For the time being, this proposal suggests throwing an error _unless_ `-D allow-untyped-meta` is defined.
+For the time being, this proposal suggests printing an error for each metadata entry that could not be typed (no metadata function could be found for its name/path) ONLY IF within a module that meets the following conditions:
```

Comment:
I still don't think it's ok as default behavior.

Reply:
Yeahhh, idk. This is mostly meant to be unresolved until the main question of whether `@.` or `@CapitalLetter` or some sort of new indicator should be used. Idk, like with a lot of things not married to it, just trying to throw out solutions. 🤔